templates/liquid/Measure.liquid

```diff
@@ -480,11 +480,11 @@
                             {% endfor %}
                         </ul>
                     {% endif %}
-                    {% if Measure.relatedArtifact.where(type = 'depends-on').exists() and
+                    {% if Measure.relatedArtifact.display.where(type = 'depends-on').exists() and
```

Comment:
You can't invoke at the `display` level here since `type` isn't an element of `display`, we need to still look at relatedArtifact.where(type = 'depends-on') but use the `display` element if it's present to provide a human-readable label for the output